usr/share/rear/conf/default.conf

```diff
@@ -1329,22 +1329,47 @@ PXE_CONFIG_URL=
 PXE_CONFIG_PREFIX=rear-
 #
 # TFPT IP server. needed to create grub menu when PXE_CONFIG_GRUB_STYLE=y
-# If not set, we gonna try to get TFTP IP from PXE_TFTP_URL
+# If not set, we gonna try to get TFTP IP from PXE_TFTP_UPLOAD_URL
 PXE_TFTP_IP=
 #
 # where should we put the TFTP files ? (legacy way)
 PXE_TFTP_PATH=/var/lib/rear/output
 # where should we put the TFTP files ? (URL style)
-PXE_TFTP_URL=
+#PXE_TFTP_URL: deprecated as of 2.8. Use PXE_TFTP_UPLOAD_URL
+PXE_TFTP_UPLOAD_URL=
 #
 # prefix for PXE files, e.g. the hostname
 PXE_TFTP_PREFIX=$HOSTNAME.
 #
+# Optional support for downloading kernel and initrd by HTTP, thus making
+# downloads considerably faster.
+# First, you have to use lpxelinux.0 instead of pxelinux.0, the former supports
+# HTTP downloads. It's compatible, just replace pxelinux.0 in your TFTP
+# directory or set bootfile-name/filename to lpxelinux.0 in your DHCP server.
+# PXE_CONFIG_GRUB_STYLE must NOT be set to yes.
+# PXE_CONFIG_URL must be set and point to your pxelinux.cfg directory using nfs or any other mountable scheme.
+# PXE_TFTP_UPLOAD_URL must still be set as it provides all files still being downloaded by TFTP.
+# PXE_HTTP_UPLOAD_URL must be set and point to your web server root directory using nfs or any other mountable scheme.
+# PXE_HTTP_DOWNLOAD_URL must be set and point to your web server root directory using http.
+# PXE_HTTP_DOWNLOAD_URL must point to the same directory as PXE_HTTP_UPLOAD_URL.
+# You may set OUTPUT_PREFIX_PXE, but keep in mind it will be added to
+# PXE_TFTP_UPLOAD_URL, PXE_HTTP_UPLOAD_URL and PXE_HTTP_DOWNLOAD_URL.
+# Example:
+#    OUTPUT=PXE
+#    PXE_CONFIG_GRUB_STYLE=
+#    OUTPUT_PREFIX_PXE=rear_kernels/$HOSTNAME
+#    PXE_CONFIG_URL=nfs://tftp-server/var/lib/tftpboot/pxelinux.cfg
+#    PXE_TFTP_UPLOAD_URL=nfs://tftp-server/var/lib/tftpboot
+#    PXE_HTTP_UPLOAD_URL=nfs://web-server/var/www/html
+#    PXE_HTTP_DOWNLOAD_URL=http://web-server
+PXE_HTTP_UPLOAD_URL=
+#
 # Optional HTTP download source for PXE (URL style)
-# for example PXE_HTTP_URL="http://pxe-over-http-srv:7777"
+# for example PXE_HTTP_DOWNLOAD_URL="http://pxe-over-http-srv:7777"
```

Comment:
Can't the same PXE_HTTP_DOWNLOAD_URL example be used here
as it was already shown above in default.conf
to provide more consistent information to the user.
E.g. why is a port specified here but not above?

Reply:
Fixed with 0a3b81a221de141de601ec4b9c85c9f757372af9